.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+jobs:
+
+  check_labels:
+    name: Check labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          one_of: bug,documentation,enhancement,feature,security,:test,>upgrade
```

Comment:
That's a good point. With the current mechanisms, we do it manually but have to sift through the PRs to understand where it belongs. This will help to simplify in the sense that it narrows down where the PR belongs and provides a heuristic to help classify. For an automated approach, we can design the solution in a way to account for the priority and we always can manually intervene if needed for a few of the changes.

Reply:
Thanks for this change. This is a good starting point towards release notes automation. 

Automation related discussion can be done on automation related PRs. Approving this one.